_rules/iframe-identical-name-equivalent-purpose-4b1c6c.md

```diff
@@ -48,6 +44,8 @@ This rule assumes that assistive technologies are exposing all `iframe` elements
 
 ## Background
 
+When determining if target elements embed the same resource, resolving the embedded resource may include any redirects that are instant.
```

Comment:
```suggestion
When determining if target elements embed the same resource, resolving the embedded resource includes any redirects that are instant.
```
I'd prefer a more direct formulation as "may include" can give the feeling that tester can choose whether they want to include it or not.